service/src/java/org/apache/hive/service/cli/session/SessionManager.java

```diff
@@ -281,6 +284,7 @@ private void initOperationLogRootDir() {
         LOG.warn("Failed to schedule cleanup HS2 operation logging root dir: " +
             operationLogRootDir.getAbsolutePath(), e);
       }
+      logManager = new OperationLogManager(this, hiveConf);
```

Comment:
Do we need the `OperationLogManager` if we do not use async log removal? Could we use `Optional` for this?

Reply:
done